build.gradle

```diff
@@ -118,7 +118,7 @@ startScripts {
 }
 
 run {
-    def confFile = "file:" + File.separator + File.separator + "$projectDir/conf/sidecar.yaml"
+    def confFile = System.getProperty("sidecar.config", "file:" + File.separator + File.separator + "$projectDir/conf/sidecar.yaml")
```

Comment:
we didn't honor the user-provided config , when I was testing locally I found out that something like this `./gradlew run -Dsidecar.config=file:///$PWD/conf/sidecar.yaml` is not working